app/helpers/foreman_ansible/smart_proxies_helper.rb

```diff
@@ -27,7 +27,13 @@ def can_update_proxy?(proxy)
     end
 
     def proxy_update_button(proxy)
-      feature = RemoteExecutionFeature.feature(:ansible_run_capsule_upgrade)
+      name = if Foreman::Plugin.find('foreman_theme_satellite').present?
```

Comment:
Do we have to actually do it? Because I can see now two temples in the run job wizard:

![Screenshot from 2023-02-02 13-40-37](https://user-images.githubusercontent.com/59385976/216327723-01cc5fb5-0a24-4208-9663-85718341a176.png)

Reply:
I was assuming this would be required so that the update button executes the right upgrade job based on the context of the installation.

Reply:
Oh yeah, I forgot about the button on proxy detail ...